src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -117,7 +117,19 @@ object SbtGit {
     gitDescribedVersion := gitReader.value.withGit(_.describedVersion).map(v => git.gitTagToVersionNumber.value(v).getOrElse(v)),
     gitCurrentTags := gitReader.value.withGit(_.currentTags),
     gitCurrentBranch := Option(gitReader.value.withGit(_.branch)).getOrElse(""),
-    gitUncommittedChanges in ThisBuild := gitReader.value.withGit(_.hasUncommittedChanges)
+    gitUncommittedChanges in ThisBuild := gitReader.value.withGit(_.hasUncommittedChanges),
+    scmInfo := {
+      val remote = """origin[ \t]+(?:git@|https?\:\/\/)([^:\/]+)[:\/](.*)\.git[ \t]+\(fetch\)""".r
+      
+      Process("git remote -v").lines_!.collect {
```

Comment:
OK, then how about using `git ls-remote --get-url origin` added in https://github.com/git/git/commit/45781adb9a89c0c47f61ccf8062be26b86a38a54 and released in v1.7.5, which will also honor `url.<base>.insteadOf`.